### ISSUE TITLE
results reproduction?

### ISSUE THREAD

**Issue by hedes1992** [dingjiansw101/AerialDetection#9]
Thanks for your excellent work and code.

I have some question about the results evaluation and reproduction
1. Does the mAP is the results of test set from evaluation server of DOTA website?
    I use the downloaded roi-transformer model and inference on test set(subSize=1024, gap=512), and get the mAP=0.7341825026925208. Meanwhile the mAP on valset is 0.8287
2. Does the official model is trained on train+val set?
    I train the roi-transformer model on train set(subSize=1024, gap=200), not contain val set. And the mAP on val set and test set is 0.7213, 0.7111493312926

**Comment by hedes1992** [comment]
> Thanks for your excellent work and code.
> 
> I have some question about the results evaluation and reproduction
> 
> 1. Does the mAP is the results of test set from evaluation server of DOTA website?
>    I use the downloaded roi-transformer model and inference on test set(subSize=1024, gap=512, rate=1.0), and get the mAP=0.7341825026925208(where
>    [model_zoo](https://github.com/dingjiansw101/AerialDetection/blob/master/MODEL_ZOO.md)) give 73.76 on OBB task.
>    Meanwhile the mAP on valset is 0.8287
> 2. Does the official model is trained on train+val set?
>    I train the [roi-transformer model](https://github.com/dingjiansw101/AerialDetection/blob/master/configs/DOTA/faster_rcnn_RoITrans_r50_fpn_1x_dota.py) on train set(subSize=1024, gap=200, rate=1.0), not contain val set. And the mAP on val set and test set(subSize=1024, gap=512, rate=1.0) is 0.7213, 0.7111493312926

I have trained some models of roi-transformer, and submit the results on testset to evaluation server. The AP are similar to the released APs.
I some modifications like the following:
1. I just use 4 imgs for each gpu on 4 gpus, the result AP is 0.7569
2. I set the bbox which difficult value is 1 or 2 to crowd in json file of training set, the result AP is 0.7338

**Comment by dingjiansw101** [comment]
Yes, the official model is trained on train+val set. The results are from the evaluation server.
